### PR TITLE
CSSkit

### PR DESCRIPTION
**Pull request by trisketz**
Update CSS stylings for landing pages to match new desgin. Update FMN component stylings for upcoming fmns and featured products.

### CODE REVIEW

File: app/components/com_fmns/site/assets/css/fmns.css [hunk 1]
```diff
@@ -75,8 +75,8 @@ body {
 }
 
 .header .bcg-fmn {
-  background: url("/app/components/com_fmns/site/assets/img/art-board-network.jpg");
-  background-image: linear-gradient(to bottom, rgba(0, 0, 0, 0.7) 0%, rgba(0, 0, 0, 0.7) 100%), url("/app/components/com_fmns/site/assets/img/art-board-network.jpg");
+  background: url("/app/site/media/images/art-board-network.jpg");
```
Comment (mdlama) [inline app/components/com_fmns/site/assets/css/fmns.css]: Just curious - better to have images in media gallery?

File: app/components/com_fmns/site/assets/css/fmns.scss [hunk 2]
```diff
@@ -1,5 +1,5 @@
-@import 'variables';
-@import 'mixins';
+@import '/Users/jenny/hubzero-vagrant/public/app/templates/bmc/css/pages/variables.scss';
```
Comment (mdlama) [inline app/components/com_fmns/site/assets/css/fmns.scss]: Heads up - this is pointing to a local directory on your vagrant box!

File: app/components/com_fmns/site/assets/css/fmns.css (hunk 1 above)
Comment (mdlama) [reply app/components/com_fmns/site/assets/css/fmns.css]: If better in the media gallery (I'm fine either way), do you mind performing a `git rm` on the images in the component?

Comment (trisketz) [reply app/components/com_fmns/site/assets/css/fmns.css]: I'm going to say to leave them in the component. I'll update the path in the css. With the exception of a couple of images, most of these won't be global.

**Comment by trisketz** [comment]
I pushed a commit to fix the file pathnames in the css. I think you should see the changes?

**Pull request merged**